src/components/EmptyStateAvatars.js

```diff
@@ -0,0 +1,80 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import {Image, View} from 'react-native';
+import styles from '../styles/styles';
+import Text from './Text';
+import ChatCustomAvatar from './ChatCustomAvatar';
+import ActiveRoomAvatar from '../../assets/images/avatars/room.svg';
+import CONST from '../CONST';
+
+const propTypes = {
+    /** Array of avatar URL */
+    avatarImageURLs: PropTypes.arrayOf(PropTypes.string),
+
+    /** Whether this avatar is for an custom room */
+    isCustomChatRoom: PropTypes.bool,
+
+    /** Whether this avatar is for an custom room */
+    CustomChatRoomIcon: PropTypes.func,
+
+};
+
+const defaultProps = {
+    avatarImageURLs: [],
+    isCustomChatRoom: false,
+    CustomChatRoomIcon: ActiveRoomAvatar,
+};
+
+const EmptyStateAvatars = ({avatarImageURLs, isCustomChatRoom, CustomChatRoomIcon}) => {
+    if (!avatarImageURLs.length) {
+        return null;
+    }
+
+    if (avatarImageURLs.length === 1 || isCustomChatRoom) {
+        return (
+            <ChatCustomAvatar
+                source={avatarImageURLs[0]}
+                imageStyles={[styles.avatarLarge]}
+                isCustomChatRoom={isCustomChatRoom}
+                CustomChatRoomIcon={CustomChatRoomIcon}
+            />
+        );
+    }
+
+    return (
+        <View pointerEvents="none">
+            <View style={[styles.flexRow, styles.wAuto, styles.ml3]}>
+                {avatarImageURLs.map((val, index) => {
+                    if (index <= 3) {
```

Comment:
I thought you changed all 3 to `CONST.REPORT.MAX_PREVIEW_AVATARS`. Please update this as well.